static/app/actionCreators/guides.tsx

```diff
@@ -56,6 +62,27 @@ export function recordFinish(guide: string, orgId: string | null) {
     },
   });
 
+  let tour;
+  switch (guide) {
+    case 'sidebar_v2':
+      tour = 'tabs';
+      break;
+    case 'issues_v3':
+      tour = 'issues';
+      break;
+    case 'release-details_v2':
+      tour = 'releases';
+      break;
+    case 'transaction_details_v2':
+      tour = 'performance';
+      break;
+    default:
+  }
+
+  if (tour) {
```

Comment:
@roggenkemper should we also check that we are in demo mode?

Reply:
yup! will add